packages/web/src/form/form.js

```diff
@@ -117,19 +117,21 @@ const FormError = ({
       {messages && (
         <div className={wrapperClassName} style={wrapperStyle}>
           <p className={titleClassName} style={titleStyle}>
-            {rootMessage}
+            {rootMessage !== '' ? rootMessage : 'Something went wrong.'}
```

Comment:
We're sure that `rootMessage` will always be at least empty string at this point? It's set too `null` originally, and assuming there is a GraphQL or Network error it will be set to those errors...

Reply:
This is in case GraphQL/Apollo sends back errors but doesn't include a message value which is the exact scenario you were seeing when you got a blank FormError component. For instance, if you don't include a message in `throw new AuthenticationError()` then you'll still get an error object response but the message value (which gets assigned to rootMessage) is a blank string. There are also a further number of errors Apollo could send back without a message ([see here](https://github.com/apollographql/apollo-server/blob/4fb348102e1248f78f6df5c70090147c5ceb7300/packages/apollo-server-errors/src/index.ts#L172)) and, presently, we don't have a way of sorting them out and returning a coherent message. I'd like to take a stab at fixing that soon but I wanted to take care of this instance first as it's an error that might be more common now that auth has been implemented.